views/index.ejs

```diff
@@ -70,6 +70,12 @@
   <p class="error-notice">If you find any errors in feature detection, please <a href="https://github.com/foolip/mdn-bcd-collector/issues/new"><span class="mdi mdi-alert-circle-outline"></span> file an issue</a> in the GitHub repository.</p>
 </div>
 
+<div id="privacy-notice" class="paper">
+  <h2><span class="mdi mdi-alert"></span> Privacy Notice</h2>
+  <p>This website is designed to test as many browser features as possible. This includes interfaces containing personal information, such as the microphone, camera, geolocation, etc. All of this information is kept local and is not sent to a server nor included in results.</p>
+  <p>Other than the browser's user agent string, which is used to identify the browser and browser version tests are run on, all data is anonymous.</p>
```

Comment:
I'm not sure about this bit, the UA string isn't personally identifying. Maybe just say that we store the UA string and why?

Reply:
I recalled hearing that a browser's UA string is considered personal information, but reading the data again I now see that it's only considered so when paired with an IP address.  Will fix this right away!